google-cloud-bigquery/pom.xml

```diff
@@ -116,6 +116,16 @@
     <dependency>
       <groupId>org.apache.arrow</groupId>
       <artifactId>arrow-memory-netty</artifactId>
+      <!--
+      Due to JDK8 requirement, we are forced to use arrow version below v18
+      which has io.netty:netty-common a dep. and its vulnerabilities
+      -->
+      <exclusions>
+        <exclusion>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-common</artifactId>
```

Comment:
You need to re-declare the dependency with a good version.

![image](https://github.com/user-attachments/assets/56c64e9f-654f-4620-894b-213beeb272e5)

From this example https://stackoverflow.com/questions/9119055/how-to-exclude-maven-dependencies